emloop/cli/eval_fn.py

```diff
@@ -34,8 +34,12 @@ def evaluate(model_path: str, eval_target: str, config_path: Optional[str], cl_a
     try:
         if delete_dir:
             print_delete_warning()
-        model_dir = path.dirname(model_path) if not path.isdir(model_path) else model_path
-        config_path = find_config(model_dir if config_path is None else config_path)
+        if model_path.endswith('.yaml'):
+            config_path = model_path
```

Comment:
this is a very weird assignment... I guess we need to discuss this in person?

Reply:
I know it's weird. The problem occurs when I want to set config file instead of model_path to emloop eval. The arguments are first parsed here: https://github.com/iterait/emloop/blob/dev/emloop/cli/args.py#L45 and then passed to evaluate function here: https://github.com/iterait/emloop/blob/dev/emloop/entry_point.py#L63. I was also thinking about setting the correct config_path here https://github.com/iterait/emloop/blob/dev/emloop/entry_point.py#L63.